python/ray/runtime_env/runtime_env.py

```diff
@@ -263,6 +267,9 @@ class MyClass:
         config: config for runtime environment. Either
             a dict or a RuntimeEnvConfig. Field: (1) setup_timeout_seconds, the
             timeout of runtime environment creation,  timeout is in seconds.
+        image_uri: URI to a container image. The Ray worker process runs
+            in a container with this image. You can't use this parameter with other fields
+            of runtime_env.
```

Comment:
can you specify what happens when the image cannot be found? does it raise an exception?

Reply:
RuntimeError: {"message":"failed to pull image after 5 retries: containerd client pull: failed to resolve reference "docker.io/<image_uri>": docker.io/<image_uri>: not found"}